readme.md

```diff
@@ -519,6 +519,7 @@
 - [Embedded and IoT Security](https://github.com/fkie-cad/awesome-embedded-and-iot-security#readme)
 - [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Regulation on data protection and privacy for all individuals within EU.
 - [DevSecOps](https://github.com/TaptuIT/awesome-devsecops#readme) - Integration of security practices into [DevOps](https://en.wikipedia.org/wiki/DevOps).
+- [Reconnaissance and Footprinting](https://github.com/nahberry/awesome-recon-tools) - Tools for reconnaissance and footprinting. 
```

Comment:
Please follow the guidelines for the description. You're also missing `#readme`.